recipe/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "snowflake_legacy" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 37096bb0020a52d641683ac8a705122dcae4b79b45ad3289d115dd82ba102af5
+  patches:
+    - patches/0001-fix-pyproject-paths.patch
+
+build:
+  number: 0
+  skip: true  # [py<38 or py>311]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  build:
+    - patch       # [not win]
+    - m2-patch    # [win]
+  host:
+    - python
+    - hatchling
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - snowflake
+    - snowflake._legacy
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pip check
+    - pytest tests
```

Comment:
I would add a test for the existence of the pth file.

Reply:
done, thank you for the suggestion!